src/App.tsx

```diff
@@ -11,7 +12,7 @@ const fetchMoreIntervalSeconds = 10;
 
 const mapState = (state: any) => ({
   blokchain: state.tezos.blocks,
-  loader: state.loader
+  loader: state.loader > 0
```

Comment:
`loader > 0`? Trochę dziwnie to wygląda - nie mogę się domyśleć o co tu chodzi 😜 Do tego `loader` zdaje się być booleanem więc może lepiej było by go nazwać np `showLoader` żeby było z daleka widać co tam może być.

Reply:
Zamysl jest taki, zeby miec globalny stan dla loadera.

Kiedy jest sytuacja, w ktorej leca 2 asynchroniczne requesty po dane( np 1 do tezosa, 2 do ethereum) i jeden z nich sie wczyta, to odrazu ustawia flage pokazujaca loader na false przez co ten drugi request, ktory sie jeszcze nie wczytal nie korzysta z loadera.

Byly 2 rozwiazania, trzymac flage `isLoading` w kazdym reducerze, ktory wczytuje te dane i na jej podstawie pokazywac loader, lub zrobic z loadera taki counter, ktory trzyma stan ile rzeczy sie wczytuje.
Domyslnie 0, czyli ma sie nie pokazywac. Jesli jest wieksze niz 0 to ma sie pokazac.